src/bin/initdb/initdb.c

```diff
@@ -3490,6 +3492,9 @@ main(int argc, char *argv[])
 			case 1003:
 				n_buffers = parse_long(optarg, true, "shared_buffers");
 				break;
+			case 1004:
+				setenv("GP_PRELOAD_LIBRARIES", optarg, true);
```

Comment:
Can we pass the flag via command line options? Mutating the global environment seems not a good idea. Is there any specific reason for that.